.travis.yml

```diff
@@ -12,7 +12,7 @@ before_install:
 
 script:
     - docker run --rm -it -v `pwd`:$AULA_SOURCE $AULA_IMAGE /bin/sh -c "$AULA_SOURCE/.travis/docker-hlint.sh"
-    - docker run --rm -it -v `pwd`:$AULA_SOURCE $AULA_IMAGE /bin/sh -c "$AULA_SOURCE/.travis/docker-build.sh --qc-max-success 1000 --qc-max-size 150"
+    - docker run --rm -it -v `pwd`:$AULA_SOURCE $AULA_IMAGE /bin/sh -c "$AULA_SOURCE/.travis/docker-build.sh --qc-max-success 100 --qc-max-size 100"
```

Comment:
We should keep the size parameter at 150. In some cases errors in paths test are not found using the default max-size 100.